doc/tutorial/climada_util_calibrate.ipynb

```diff
@@ -0,0 +1,3101 @@
+       "464    1.0   0.002204  2017-0383DOM                 True  False  False   True   \n",
+       "465    1.0   0.000271  2017-0383PRI                 True  False  False   True   \n",
+       "\n",
+       "     Slide  Other  OtherThanSurge  \n",
+       "326  False  False            True  \n",
+       "331  False  False            True  \n",
+       "334  False  False            True  \n",
+       "339   True  False            True  \n",
+       "340  False  False           False  \n",
+       "344  False  False           False  \n",
+       "345  False  False            True  \n",
+       "346  False  False            True  \n",
+       "352   True  False            True  \n",
+       "359  False  False           False  \n",
+       "365  False  False           False  \n",
+       "369  False  False           False  \n",
+       "406  False  False            True  \n",
+       "427  False  False            True  \n",
+       "428  False  False           False  \n",
+       "429  False  False            True  \n",
+       "437  False  False            True  \n",
+       "451   True  False            True  \n",
+       "456  False  False           False  \n",
+       "457  False  False           False  \n",
+       "458  False  False            True  \n",
+       "459  False  False           False  \n",
+       "460  False  False            True  \n",
+       "462  False  False           False  \n",
+       "463   True  False            True  \n",
+       "464   True  False            True  \n",
+       "465  False   True            True  \n",
+       "\n",
+       "[27 rows x 22 columns]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "from climada.util.constants import SYSTEM_DIR\n",
+    "\n",
+    "emdat = pd.read_csv(SYSTEM_DIR / \"tc_impf_cal_v01_EDR.csv\")\n",
+    "emdat_subset = emdat[(emdat.cal_region2 == \"NA1\") & ((emdat.year >= 2010))]\n",
+    "emdat_subset"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each entry in the database refers to an economic impact for a specific country and TC event.\n",
+    "The TC events are identified by the ID assigned from the International Best Track Archive for Climate Stewardship (IBTrACS).\n",
+    "We now want to reshape this data so that impacts are grouped by event and country.\n",
+    "\n",
+    "To achieve this, we iterate over the unique track IDs, select all reported damages associated with this ID, and concatenate the results.\n",
+    "For missing entries, `pandas` will set the value to NaN.\n",
+    "We assume that missing entries means that no damages are reported (this is a strong assumption), and set all NaN values to zero.\n",
+    "Then, we transpose the dataframe so that each row represents and event and each column states the damage for a specific country.\n",
```

Comment:
very small typo: "each row represents _and_ event" should be "each row represents _an_ event"

Reply:
Fixed in 472d0c56fc00891f24255ae58f7387ed2b2589d8